app.js

```diff
@@ -10,9 +10,7 @@ const dbConfig = require('./database/db');
 var cookieparser =require('cookie-parser');
 const csurf = require('csurf');
 var multer = require('multer');
-var upload = multer({
-  limits: { fileSize: 1024*1024 }
-});
+var upload = multer();
```

Comment:
#### Automated PR Comment From coverity
<!-- 5d028e76497918c7e3dee6abe7e5ae6f -->
<!-- Comment managed by Git Commenter, do not modify! -->
<!-- Comment managed by coverity-github-pr-commenter, do not modify!
5d028e76497918c7e3dee6abe7e5ae6f
-->
# Coverity Issue - File upload with insecure storage
_Low CWE-921_
The `multer` middleware is configured to use memory for storing file uploads rather than persisting them on disk.
It is possible to cause a denial of service attack by uploading a large number of files or one big file into the server's memory.
## How to fix
Avoid using `MemoryStorage` in production environments. Configure `multer` to store file uploads on a disk by setting the `storage` option to the built-in `diskStorage` module or configuring the `dest` option as a folder on disk where the files are stored.